### PR TITLE
:bug: Make tip of the week snippet optional

### PR DESCRIPTION
**Pull request by prateek3255**

### CODE REVIEW

**Comment by prateek3255** [comment]
Also added draft release here - https://github.com/scriptified/scriptified/releases/edit/untagged-d28c85f876eba98d9e77

File: scripts/issueEmailGenerator.js [hunk 1]
```diff
@@ -137,23 +141,6 @@ ${tool.description}
...
-## ${currentIssue.devOfTheWeek.name}
-${currentIssue.devOfTheWeek.bio}
-
-${Object.keys(currentIssue.devOfTheWeek)
-  .filter(key => PROFILE_KEYS.includes(key) && currentIssue.devOfTheWeek[key] !== null)
-  .map(profile => `[${PROFILE_TYPES[profile]}](${currentIssue.devOfTheWeek[profile]})`)
-  .join(' | ')}
-
-___
-
```
Comment (gupta-ji6) [inline scripts/issueEmailGenerator.js:156]: let's make it conditional instead of removing.

**Review by gupta-ji6 — CHANGES_REQUESTED** [review]

File: scripts/issueEmailGenerator.js (hunk 1 above)
Comment (gupta-ji6) [inline scripts/issueEmailGenerator.js:156]:
```
${
  currentIssue.devOfTheWeek
    ? `
# Dev of the Week

<img alt="${currentIssue.devOfTheWeek.name}" src="${getAssetURL(
        currentIssue.id,
        currentIssue.devOfTheWeek.profileImg
      )}" style="width:200px;"/>

## ${currentIssue.devOfTheWeek.name}
${currentIssue.devOfTheWeek.bio}

${Object.keys(currentIssue.devOfTheWeek)
  .filter(
    (key) =>
      PROFILE_KEYS.includes(key) && currentIssue.devOfTheWeek[key] !== null
  )
  .map(
    (profile) =>
      `[${PROFILE_TYPES[profile]}](${currentIssue.devOfTheWeek[profile]})`
  )
  .join(' | ')}

___
`
    : null
}

```

Comment (prateek3255) [reply scripts/issueEmailGenerator.js:156]: ✅

**Review by gupta-ji6 — APPROVED** [review]

**Pull request merged**